app/controllers/api/v1/users_controller.rb

```diff
@@ -14,8 +17,14 @@ def create
 
       private
 
+      def find_user
+        @user = User.find_by!(username: params[:_username])
+      rescue ActiveRecord::RecordNotFound
+        render json: { errors: 'User not found' }, status: :not_found
+      end
+
       def user_params
-        params.require(:user).permit(:email, :password, :first_name, :last_name)
+        params.permit(:email, :password, :first_name, :last_name)
```

Comment:
Es método `require` debería estar para lanzar la excepción.
```suggestion
        params.require(:user).permit(:email, :password, :first_name, :last_name)
```

Reply:
Sugiero quitar estos cambios, están contemplados en otro ticket.